Content.Server/Utility/GridTileLookupHelpers.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Content.Shared.Maps;
+using Robust.Server.GameObjects.EntitySystems.TileLookup;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Map;
+
+namespace Content.Server.Utility
+{
+    public static class GridTileLookupHelpers
+    {
+        /// <summary>
+        ///     Helper that returns all entities in a turf very fast.
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static IEnumerable<IEntity> GetEntitiesInTileFast(this TileRef turf)
+        {
+            var gridTileLookup = EntitySystem.Get<GridTileLookupSystem>();
```

Comment:
Given that this is "Fast" and you are even inlining it you might as well allow this system to be passed as a nullable parameter.

Reply:
true, done